### ISSUE TITLE
Cannot load module "CoreModule"

### ISSUE THREAD

**Issue by Guztaver** [gorilla-devs/GDLauncher-Carbon#316]
# What is happening?

So, after some time my launcher just not starts anymore, and I don't have a clue what is happening. I already make the things that is supposed to quick fix it, but not solved. I even uninstalled and installed the launcher, nothing still hapenning.

```
User data: C:\Users\Gustavo\AppData\Roaming\gdlauncher_carbon
Initial runtime path: C:\Users\Gustavo\AppData\Roaming\gdlauncher_carbon\data
Runtime path: C:\Users\Gustavo\AppData\Roaming\gdlauncher_carbon\data
```
```
Logs path: C:\Users\Gustavo\AppData\Roaming\gdlauncher_carbon\data\__gdl_logs__ loaded default logger directives, to override, set `RUST_LOG` env var RUST_LOG="debug,carbon_app=trace,hyper::client::pool=warn,reqwest::connect=warn,hyper::proto::h1::conn=warn,hyper::proto::h1::io=warn,hyper::proto::h1::decode=warn,quaint::connector::metrics=warn,hyper::client::connect::http=warn,hyper::client::connect::dns=warn,quaint::pooled::manager=warn,rustls::client::hs=warn,rustls::client::tls13=warn,h2::client=warn,rustls::client::common=warn,h2::codec::framed_read=warn,h2::codec::framed_write=warn,h2::proto::settings=warn,tungstenite::protocol=warn,mobc=trace"

thread 'main' panicked at C:\Users\circleci\project\crates\iridium\src\lib.rs:62:67: called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Builder, source: Os { code: -2146893809, kind: Uncategorized, message: "O objeto já existe." } } stack backtrace: 0: 0x7ff69dcd081a - 1: 0x7ff69d578c9d - 2: 0x7ff69dcb3dd1 - 3: 0x7ff69dcd2c9a - 4: 0x7ff69dcd25ff - 5: 0x7ff69dcd2161 - 6: 0x7ff69dcd3013 - 7: 0x7ff69dcd2eb5 -

8: 0x7ff69dcd2df9 - 9: 0x7ff69dcd2de4 - 10: 0x7ff69dee3197 - 11: 0x7ff69dee3743 - 12: 0x7ff69d35245c - 13: 0x7ff69d36bb62 - 14: 0x7ff69d368040 - 15: 0x7ff69d35f7c8 - 16: 0x7ff69ce6e4b1 - 17: 0x7ff69ce7321b - 18: 0x7ff69decb65c - 19: 0x7ff967ba257d - BaseThreadInitThunk 20: 0x7ff96892aa58 - RtlUserThreadStart
```
![image](https://github.com/gorilla-devs/GDLauncher-Carbon/assets/39814943/c95cf64b-3d76-4abb-8ac4-741e7ea12739)

# OS
- Platform: `Windows 11 Pro`

**Comment by Guztaver** [comment]
And I already asked help on Discord, still not recieving any response...

**Comment by Xelbayria** [comment]
be patient. It can take some times for someone to respond.

**Comment by Meerder** [comment]
I found a solution to the coremodule problem if anyone's still having issues
**ALWAYS REMEMBER TO BACK UP YOUR DATA**
1: Close in task manager (Sometimes the game is running in the background and the core module file is still running with it in the background, force closing allows it to reboot coremodule with the foreground app)
**2: Update your launcher** (Running an old version of gdlauncher is like running an old version of the Minecraft launcher, it won't let you load the launcher let alone the game, uninstall and download the up to date version of gd Carbon)
**3: Close GD and open vanilla Minecraft Launcher** (Sometimes the coremodule has a hard time sourcing and extracting all of the information for Minecraft instances, textures and other important elements of the game and configuring it for another JavaScript that is GD. Quit GD, open Minecraft launcher until it hits the main menu _{ya don't gotta open mc}_. Quit Minecraft launcher and repeat the first 2 steps if you haven't for essentially a clean launcher slate)
Hopes this helps

**Comment by blarfoon** [comment]
Core module can have all different kind of issues.
For this specific one, the solution would be to update to the latest GDL version (2.0.9).

**Comment by Meerder** [comment]
> Core module can have all different kind of issues. For this specific one, the solution would be to update to the latest GDL version (2.0.9).

This is for GD Carbon, not Legacy, also legacy does not work for me so I wouldn't know about switching, just fixing an issue with carbon

**Comment by blarfoon** [comment]
Yes I know. The issue above is related to OpenSSL which we no longer use in 2.0.9
